.eslintrc.js

```diff
@@ -103,37 +104,54 @@ module.exports = {
     'react/jsx-space-before-closing': 'error',
     'react/jsx-uses-react': 'error',
     'react/jsx-uses-vars': 'error',
+    'react/no-comment-textnodes': 'error',
     'react/no-danger': 'error',
+    'react/no-deprecated': 'error',
     'react/no-did-mount-set-state': 'error',
     'react/no-did-update-set-state': 'error',
     'react/no-direct-mutation-state': 'error',
     'react/no-multi-comp': 'off', // Wishlist, one day
-    'react/no-unknown-property': 'error',
+    'react/no-render-return-value': 'error',
     'react/no-is-mounted': 'error',
+    'react/no-unknown-property': 'error',
     'react/prefer-arrow-callback': 'off', // Wishlist, one day
     'react/prefer-es6-class': 'error',
     'react/prop-types': 'error',
     'react/react-in-jsx-scope': 'error',
     'react/require-extension': 'error',
+    'react/require-render-return': 'error',
     'react/self-closing-comp': 'error',
     'react/sort-comp': 'error',
     'react/sort-prop-types': 'error',
     'react/wrap-multilines': 'error',
-    'react/jsx-indent': ['error', 2],
-
-    // React Disabled
-    'react/jsx-no-bind': 'off',
-    'react/jsx-no-literals': 'off',
-    'react/jsx-sort-props': 'off',
-    'react/no-set-state': 'off',
 
-    // Material-UI
     'material-ui/docgen-ignore-before-comment': 'error',
 
     'mocha/handle-done-callback': 'error',
     'mocha/no-exclusive-tests': 'error',
     'mocha/no-global-tests': 'error',
     'mocha/no-pending-tests': 'error',
     'mocha/no-skipped-tests': 'error',
+    'mocha/valid-suite-description': 'off',
+    'mocha/valid-test-description': 'off',
+
+    'strict': 'off',
+    'no-case-declarations': 'off',
+
+    'react/jsx-key': 'off',
+    'react/jsx-no-bind': 'off',
+    'react/jsx-no-literals': 'off',
+    'react/jsx-no-target-blank': 'off',
+    'react/jsx-sort-props': 'off',
+    'react/no-set-state': 'off',
+    'react/no-string-refs': 'off',
```

Comment:
> refs are deprecated

Well, that's not yet official, but it's close. What about using `warn` and add `--quite` in the script to no block the CI tool?